doc/content/getting-started/migrating-from-v2/major-changes.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Major Changes in The Things Stack
+weight: 10
+---
+
+Before getting started, we will discuss major breaking changes between {{% ttnv2 %}} and {{% tts %}}, along with some guidelines to make the migration process easier to manage.
+
+### Architecture
+
+Behind the scenes, {{% tts %}} is using a new architecture. See {{< ref "/reference/components" >}} for a complete overview of the different components of {{% tts %}}.
+
+### LoRaWAN support
+
+{{% tts %}} requires the LoraWAN MAC Protocol Version and Regional Parameters(LoraWAN PHY version) to be set per device. These default to `MAC_V1_0_2` and
+`MAC_PHY_V1_0_2_REV_B` for devices imported from {{% ttnv2 %}}.
+
+The LoraWAN MAC settings are configurable per end device. This means that all end devices need a frequency plan. Above, we used the `EU_863_870` frequency plan, but this needs to be replaced with the frequency plan corresponding to your region. A list of supported frequency plan IDs in [the lorawan-frequency-plans Github repository](https://github.com/TheThingsNetwork/lorawan-frequency-plans/blob/master/frequency-plans.yml).
+
+Furthermore, {{% tts %}} brings full support for all LoraWAN versions (1.0.0 through 1.1.0), as well as Class B and Class C devices.
+
+### Gateway coverage
+
+The Packet Broker enables peering between networks, so traffic received by one network (e.g. the public community network) but intended for a different network ({{% tts %}}) can be forwarded to and from that network.
+
+With Packet Broker enabled (which is not discussed here), you can receive traffic on {{% tts %}} without the need to configure any of your gateways (since  traffic will be automatically forwarded from the public community network).
```

Comment:
You might want to refer to the Peering guide / concept

Reply:
I am not sure which one that is.

Reply:
There's one under Integrations now

Reply:
It is under the `develop` branch and this guide is based on master, so it is not available yet. Link will work after these are merged, is this enough? Should I rebase to develop?

Reply:
Yeah retarget to `develop`; we're releasing this anyway

Reply:
This is now up-to-date